VideoLocker/src/main/java/org/edx/mobile/logger/Logger.java

```diff
@@ -30,8 +36,28 @@ public Logger(String tag) {
         this.tag = tag;
     }
 
+    /**
+     * Prints the stack trace for the given throwable instance for debug build.
+     * @param ex
+     */
     public void error(Throwable ex) {
+        error(ex, false);
+    }
+
+    /**
+     * Prints the stack trace for the given throwable instance for debug build.
+     * Also, submits the crash report (only for release build) to Fabric if submitCrashReport is true.
+     * @param ex
+     * @param submitCrashReport
+     */
+    public void error(Throwable ex, boolean submitCrashReport) {
         LogUtil.error(this.tag, "", ex);
+
+        if (submitCrashReport
+                && !LogUtil.ISDEBUG
+                && Config.getInstance().getFabricConfig().isEnabled()) {
+            Crashlytics.logException(ex);
+        }
     }
```

Comment:
should we just check 
if ( Config.getInstance().getFabricConfig().isEnabled()) ?